src/kirby/components/list/list.component.scss

```diff
@@ -36,8 +36,20 @@ $shadow: get-elevation(2);
       margin-bottom: $outline-width;
       background-color: $background-color-hover;
     }
-  }
 
+    $list-colors: ('white', 'light');
```

Comment:
Is there an other way to get these two colors then this? I can see an issue if somewhere down the road the names `white` or `light` changes.

Reply:
Well, I could get it directly from the $system-colors map, which will add some type safety to this, but might make up for some weird code in a scss file...